moped-editor/src/views/projects/projectView/ProjectComponents/utils/useCreateComponent.js

```diff
@@ -38,7 +37,11 @@ const createReducer = (state, action) => {
         showCreateDialog: false,
       };
     case "store_draft_component":
-      return { ...state, draftComponent: action.payload };
+      return {
+        ...state,
+        draftComponent: action.payload,
+        isCreatingComponent: true,
```

Comment:
I just moved switching `isCreatingComponent` to true here instead of in `start_create` because I noticed that the CTN layer sometimes showed when the create modal first appears. The CTN layer will now only show after the **Continue** button is clicked in the create modal and once the user is interacting with the map.